README.md

```diff
@@ -17,20 +17,35 @@
 **Release notes and unreleased changes can be found in the [CHANGELOG](CHANGELOG.md)**
 
 ## Requirements
-- rust v1.56 and up
+- Rust v1.56 and up
 - MacOS (x86) or Linux (x86) for sandbox tests. Testnet is available regardless
 
 ### M1 MacOS
-NOTE: Current version of workspaces does not provide support for M1 macbooks, due to some internal upgrades with wasmer. Look at using for workspaces version `0.1.1` for now until we get this settled out or check the progress of it with this [issue](https://github.com/near/workspaces-rs/issues/110).
+NOTE: Current version of `workspaces-rs` does not support use on M1 chip devices due to internal upgrades with wasmer. M1 users should use `workspaces-rs` version `0.1.1` until this problem gets resolved. Check the progress on this issue [here](https://github.com/near/workspaces-rs/issues/110).
 
-To be able to use this library on an M1 Mac, we would need to setup rosetta plus our cross compile target:
+Even with the above note, we can use `workspaces-rs` with version `0.1.1` on M1 by setting up rosetta plus our cross compile target:
 ```
 softwareupdate --install-rosetta
 rustup default stable-x86_64-apple-darwin
 ```
 
-### Note about compiling with contract
-`workspaces-rs` does not currently support compiling to WASM. So, if we are compiling this library alongside a `wasm32` target such as compiling alongside a NEAR contract, it is best if we put this dependency in `[dev-dependencies]` section to avoid any conflicts.
+### Note about compiling with NEAR Contracts
+`workspaces-rs`, the library itself, does not currently compile to WASM. So, if we were compiling it alongside a `wasm32` target, such as compiling alongside a NEAR contract:
+```toml
+# Cargo.toml
+[dependencies]
+workspaces = "*"
+near-sdk = "*"
+```
+It would throw an error when we run `cargo build --target wasm32-unknown-unknown`. Instead, we should add `workspaces-rs` into `[dev-dependencies]` for testing contracts:
+```toml
+# Cargo.toml
+[dependencies]
+near-sdk = "*"
+
+[dev-dependencies]
+workspaces = "*"
```

Comment:
The 0.2 version probably should have been included here, no reason to wildcard

Reply:
yeah, just didn't want to miss having to update the text here when we update/release past "0.2". I can change this to just `workspaces = ...` just so readers don't confuse it with valid syntax